integrations/pinterest-tag/lib/index.js

```diff
@@ -63,15 +63,27 @@ Pinterest.prototype.identify = function(identify) {
 
 Pinterest.prototype.page = function(page) {
   // If we have a category, the use ViewCategory. Otherwise, use a normal PageVisit.
+  var pinterestPageProps = {
+    name: page.name() || ''
+  };
+
+  var eventKeys = ['event_id', 'eid', 'eventID'];
```

Comment:
```suggestion
  const eventKeys = ['event_id', 'eid', 'eventID'];
```

Reply:
In this repo, Eslint is not allowing let and const variable declarations.